distr_tracing/distr_tracing_otel/distr-tracing-otel-migrating.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="dist-tracing-otel-migrating"]
+= Migrating from the {JaegerShortName} to the {OTELShortName}
+include::_attributes/common-attributes.adoc[]
+:context: dist-tracing-otel-migrating
+
+toc::[]
+
+If you are already using {JaegerName} for your applications, you can migrate to the {OTELName}, which is based on the link:https://opentelemetry.io/[OpenTelemetry] open-source project. The {OTELShortName} provides a set of APIs, libraries, agents, and instrumentation to facilitate observability in distributed systems. The OpenTelemetry Collector in the {OTELShortName} can ingest the Jaeger protocol, so you do not need to change the SDKs in your applications. Migration from the {JaegerShortName} to the {OTELShortName} requires configuring the OpenTelemetry Collector and your applications to report traces seamlessly. You can migrate sidecar and sidecarless Deployments.
```

Comment:
I would consider breaking this paragraph up, it's a lot to take in :)

Reply:
I turned it into three shorter paragraphs.